src/sentry/deletions/defaults/group.py

```diff
@@ -25,6 +25,8 @@
 
 GROUP_CHUNK_SIZE = 100
 EVENT_CHUNK_SIZE = 10000
+GROUP_HASH_CHUNK_SIZE = 10000
```

Comment:
I'm likely going to reduce this since I reduced it to 1k here: https://github.com/getsentry/sentry-options-automator/pull/5207